src/main/java/com/cloudogu/smeagol/wiki/usecase/EditPageCommandHandler.java

```diff
@@ -0,0 +1,50 @@
+package com.cloudogu.smeagol.wiki.usecase;
+
+import com.cloudogu.smeagol.Account;
+import com.cloudogu.smeagol.AccountService;
+import com.cloudogu.smeagol.wiki.domain.*;
+import de.triology.cb.CommandHandler;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import java.time.Instant;
+import java.util.Optional;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'java.util.Optional'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://ecosystem.cloudogu.com/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)